tapglue-android-sdk/src/main/java/com/tapglue/networking/TGNetworkManager.java

```diff
@@ -415,8 +424,40 @@ private void performRequest(@NonNull final TGRequest request, boolean flushIfPos
                     sendErrorToCallbacks(request.getCallback(), TGRequestErrorType.ErrorType.UNSUPPORTED_INPUT);
                 }
                 else {
```

Comment:
Please remove the else from here and use a `break` in the previous block after the `sendErrorToCallbacks` call.